src/index.html

```diff
@@ -1,17 +1,197 @@
+          <label for="chbx">I accept the term of the agreement</label>
+
+          <input
+            type="checkbox"
+            id="chbx"
+            name="accept"
+            value="Yes"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="form-set">
+        <legend>Registration</legend>
+        <div class="form-field">
+          <label for="email">E-mail:</label>
+
+          <input
+            type="email"
+            id="email"
+            name="email"
+            required
+            placeholder="email@example.com"
+          >
+        </div>
+
+        <div>
+          <label for="password">Password:</label>
+          
+          <input
+            type="password"
+            id="password"
+            name="password"
+            required
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="form-set">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">
+          <label for="radio">Do you love cats?</label>
+
+          <input
+            type="radio"
+            name="radio"
+          >
+          Yes
+          <input
+            type="radio"
+            name="radio"
+          >
+          No
+        </div>
+
+        <div class="form-field">
+          <label for="color">What is your favorite color?</label>
+
+          <input
+            type="color"
+            id="color"
+            name="color"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="goBed">What time do you go to bed?</label>
+
+          <input
+            type="time"
+            id="goBed"
+            name="name"
+            step="2"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="brandOfCars">What are your favorite brand of cars?</label>
+
+          <select name="brandOfCars" multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </div>
+
+        <div>
+          <label for="rateWork">
+            How do you rate our work?
+          </label>
+
+          <input
+            type="range"
+            id="rateWork"
+            name="rateWork"
+            min="0"
+            max="100"
+            value="0"
+          >
+        </div>
+      </fieldset>
+      
+      <fieldset class="form-set">
+        <legend>Additional info:</legend>
+
+        <div class="form-field">
+          <label for="comments">Comments:</label>
+
+          <textarea name="comments"
+            autocomplete="off">
+          </textarea>
+        </div>
+        
+        <div>
+          <label for="recom">Would you recommend us?</label>
+          <select name="recom">
+            <option value="yes">yes</option>
+            <option value="no">no</option>
+          </select>
+        </div>
+      </fieldset>
+    </form>
+    <button type="submit">Submit</button>
```

Comment:
button should be inside form
```suggestion
      </fieldset>
      <button type="submit">Submit</button>
    </form>
    
```